crates/hyperswitch_connectors/src/connectors/airwallex/transformers.rs

```diff
@@ -147,6 +147,7 @@ pub struct Browser {
     user_agent: String,
 }
 
+#[allow(dead_code)]
```

Comment:
Why is this added? 👀

Reply:
getting this warning during 
<img width="628" alt="Screenshot 2024-12-27 at 12 12 45" src="https://github.com/user-attachments/assets/6d31f281-bf9b-4dc6-9710-1d20fb4c5e52" />
 `cargo clippy `

Reply:
no, removing this